db/migrate/20201014151211_change_reasoning_to_be_text_in_deprecated_notebooks.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeReasoningToBeTextInDeprecatedNotebooks < ActiveRecord::Migration
+  def up
+    change_column :deprecated_notebooks, :reasoning, :text
```

Comment:
Went back and looked at the ticket.  Ticket talks about restricting length in Ruby.  This can't be merged without the corresponding piece otherwise the codebase is in a state where extremely large values can make it into the database (and once this is merged, it becomes part of the docker image).

Reply:
Oh shoot, I totally forgot about that. I agree, you're right, we definitely need a limit and error handling that enforces it. I'm on it.